modules/core/example/ts/avaxc-test.ts

```diff
@@ -0,0 +1,45 @@
+import { BitGo } from 'bitgo';
+const bitgo = new BitGo({
+  env: 'test',
+});
+const coin = 'tavaxc';
+const walletLabel = 'Test TAVAXC V2 Wallet - ' + (new Date()).getTime();
+const accessToken = '';
+const walletId = '';
+const walletPassphrase = '';
+
+async function createWallet() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+
+  const walletOptions = {
+    label: walletLabel,
+    passphrase: walletPassphrase,
+    walletLabel,
+    walletPassphrase,
+    enterprise: '',
+  };
+
+  const wallet = await bitgo.coin(coin).wallets().generateWallet(walletOptions);
+  const walletInstance = wallet.wallet;
+  console.log(`Wallet ID: ${walletInstance.id()}`);
+}
+
+async function sendMoneyWithHop() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+
+  const wallet = await bitgo.coin(coin).wallets().getWallet({ id: walletId });
+
+  const res = await wallet.sendMany({
+    recipients: [{
+      amount: '',
+      address: '',
+    }],
+    walletPassphrase: walletPassphrase,
+    hop: true,
+  });
+
+  console.log(res);
```

Comment:
We can log more data here. e.g. https://github.com/BitGo/BitGoJS/blob/master/modules/core/example/ts/send-wallet-transaction.ts#L38-L43